sdk-setup/src/mb2

```diff
@@ -58,9 +58,12 @@ usage() {
   $ME is aware of spectacle and will update the spec file if there is
   an obvious yaml file which is newer.
 
-  $ME build [-d] [-j <n>] [<args>]
+  $ME prep           : runs rpmbuild for %prep phase of the spec file.
+
+  $ME build [-bp|--doprep] [-d|--enable-debug] [-j <n>] [<args>]
                      : runs rpmbuild for the given spec file in the
                        given sb2 target. Produces an rpm package.
+                     : -bp    run %prep phase when building as well
```

Comment:
I prefer that single-dash options are also single-letter. Just '-p' would be OK with me.